config/internal/apiserver/default/role_ds-pipeline.yaml.tmpl

```diff
@@ -16,6 +16,13 @@ rules:
       - get
       - list
       - delete
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - get
+      - list
   - apiGroups:
```

Comment:
why is this needed? I don't see any changes to code that gets or lists secrets.

Reply:
we have to fetch the object store creds for creating the s3 client to create the presigned url